README.md

```diff
@@ -1,44 +1,428 @@
-# test-mule
+# Test Mule
 
-[not ready for use]
+Test Mule is a library that allows you to use real browsers in your Jest tests. Test Mule is focused on helping you write tests that are [as similar as possible to how users use your application](https://twitter.com/kentcdodds/status/977018512689455106). It is built on [Puppeteer](https://github.com/puppeteer/puppeteer), [Testing Library](https://testing-library.com), and [jest-dom](https://github.com/testing-library/jest-dom).
```

Comment:
There's a comment below about discussing the "Why?" earlier. After returning to the beginning of this document, it could could probably be worked into this paragraph somehow. Right now this accurately describes what Test Mule is but doesn't really spell out _why_ it is or answer the reader's likely first question: _Why should I care about this? (...among the other testing tools I already know)_.  I think the easiest way to make that connection is to speak to the audience's "pain". For inspiration, [this new js framework](https://markojs.com/) did a pretty good job of catching my attention, even though I'm **not** in the market for a new js framework. I think it worked on me because they quickly touched on pain points that I recognize.

Reply:
I pushed a change, let me know if there are any suggestions for stuff to add or change for the Why

Reply:
I spotted some changes lower down. But at the moment, nothing on line 3 speaks to _why_.  As an exercise, I recommend trying to answer this question directly in 3 sentences or less: "As a developer, why should I care about Test Mule?" (and then incorporate that above)

Another exercise that might help: "As someone who doesn't know much about testing, why should I care about Test Mule?" 

Let's chat tomorrow if what I'm nudging you to do isn't making sense.

Reply:
@emersonthis I rewrote the first couple paragraphs, mind taking another look? I think I've hit the main points of why Test Mule exists and why people should care about it, both for people who are new to browser testing, and for people who have experience with other tools.

Reply:
I like it!